oozie-to-airflow/templates/action.tpl

```diff
@@ -13,8 +13,8 @@
   See the License for the specific language governing permissions and
   limitations under the License.
  #}
-{% for op in sub_ops %}
-{{ op.rendered_template }}
+{% for task in tasks %}
+{{ task.rendered_template }}
```

Comment:
Why are we renaming this template? Even if it's needed, I'd suggest a separate commit/PR.

Reply:
A separate PR will have no reason to introduce this change. We agreed that every PR should be justified.

Reply:
I introduce unification in this PR, so that there is no duplicate code in the project.

Reply:
Ok, let's do this here. However for the future I'd suggest a separate commit explaining the unification of `fs.tpl` --> `action.tpl`.